sys/net/gnrc/network_layer/icmpv6/echo/gnrc_icmpv6_echo.c

```diff
@@ -107,4 +107,123 @@ void gnrc_icmpv6_echo_req_handle(gnrc_netif_t *netif, ipv6_hdr_t *ipv6_hdr,
     }
 }
 
+static void _fill_payload(uint8_t *buf, size_t len, uint32_t now)
+{
+    uint8_t i = 0;
+
+    if (len >= sizeof(uint32_t)) {
+        memcpy(buf, &now, sizeof(now));
+        len -= sizeof(now);
+        buf += sizeof(now);
+    }
+
+    while (len--) {
+        *buf++ = i++;
+    }
+}
+
+static bool _check_payload(const void *buf, size_t len, uint32_t now,
+                           uint32_t *triptime, int *corrupt)
```

Comment:
You do not check the return value in https://github.com/RIOT-OS/RIOT/pull/18934/files#diff-61cd6515369036b5fa6099cd4769d1cb6d71d5f21cb734862804feb281146380R224, the only instance it is called as far as I can see. So use void as return type instead?

Reply:
ah right `corrupt` used to be unsigned, now the separate return value is no longer needed